pip_executable_choco/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,28 @@
+﻿
+
+
+$ErrorActionPreference = 'Stop';
+
+
+Write-Host "Warning: Only removing pipx and path entry. Not removing pipx installed venvs. These can be found under $env:UserProfile\.local\pipx"
+
+# region: remove PATH values
+# based on https://gitlab.com/DarwinJS/ChocoPackages/-/blob/master/ec2clitools/tools/chocolateyuninstall.ps1
+$PathToRemove = "$env:USERPROFILE\.local\bin"
+
+foreach ($path in [Environment]::GetEnvironmentVariable("PATH","User").split(';'))
```

Comment:
Use our coding conventions for this file.

https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/987/PowerShell-Coding-Conventions